src/scriptworker/context.py

```diff
@@ -112,18 +112,18 @@ def credentials(self):
         if self._credentials:
             return dict(deepcopy(self._credentials))
 
-    @property
-    def task_id(self):
-        """string: The running task's taskId."""
-        if self.claim_task:
-            return self.claim_task["status"]["taskId"]
-
     @credentials.setter
```

Comment:
this is due to a mypy bug, which flags @credentials.setter as wrong unless it is right next to the @property getter.

Reply:
That's probably preferred sorting anyway, no worries :)